README.md

```diff
@@ -274,9 +279,22 @@ Notice how some of the configuration values are wrapped in angle brackets (e.g.
                 "state_timeout_in_seconds": 600,
 
                 // The name of the OIDC claim which provides the mapping of an
-                // OIDC user to an iRODS user account
+                // OIDC user to an iRODS user account.
+                // Only 'irods_user_claim' or 'user_attribute_mapping' can be
+                // declared at once.
                 "irods_user_claim": "irods_username",
 
+                // The mapping of user to the provided values. All values must
+                // be match to map an OIDC user to and iRODS user account.
```

Comment:
`"All values must be match to map an OIDC user to and iRODS user account."`?

Reply:
Brain-keyboard latency, should make sense now.